app/models/event.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  has_many :user_events, foreign_key: :attended_event_id
+  has_many :attendees, through: :user_events, foreign_key: :attendee_id
```

Comment:
- [ ] **[OPTIONAL]** I was able to create an event in the past so we need to validate the date to only become future dates.